components/3d/dao/PlaceDAO.tsx

```diff
@@ -0,0 +1,181 @@
+  MdHouse,
+  MdMusicNote,
+  MdPeople,
+  MdQuestionMark,
+  MdTrain,
+} from 'react-icons/md';
+import Place, { OrbitControlTarget } from '../entity/Place';
+
+export type PlacesDatabase = {
+  DEVESA: Place;
+  EVENTS: Place;
+  STATION: Place;
+  CATHEDRAL: Place;
+  GLOBAL: Place;
+  GIRONAJS: Place;
+  AUDITORIUM: Place;
+};
+
+const places: PlacesDatabase = {
+  GLOBAL: new Place( //GLOBAL IS DEBUG
+    'GLOBAL',
+    'Global',
+    MdQuestionMark,
+    new OrbitControlTarget(new THREE.Vector3(0, 0, 0), 10),
+    new THREE.Vector3(0, 0, 0),
+    true,
+    false,
+    'Global',
+    'Debug',
+    '',
+    null,
+    null,
+    null,
+    ['', '']
+  ),
+  GIRONAJS: new Place(
+    'GIRONAJS',
+    'places.gironajs.navbar_name',
+    MdHouse,
+    new OrbitControlTarget(new THREE.Vector3(-2.63, 0.3, 3), 5),
+    new THREE.Vector3(-2.63, 1, 3),
+    false,
+    false,
+    'places.gironajs.title',
+    'places.gironajs.subtitle',
+    'places.gironajs.description_html',
+    'https://goo.gl/maps/WJjMS189fTomSLwJ9',
+    'https://gironajs.com/en',
+    null,
+    [
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/9/99/Unofficial_JavaScript_logo_2.svg/800px-Unofficial_JavaScript_logo_2.svg.png',
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/8/86/159_Farinera_Teixidor%2C_c._Santa_Eug%C3%A8nia_42_%28Girona%29.jpg/1024px-159_Farinera_Teixidor%2C_c._Santa_Eug%C3%A8nia_42_%28Girona%29.jpg',
+    ]
+  ),
+  CATHEDRAL: new Place(
+    'CATHEDRAL',
+    'places.cathedral.navbar_name',
+    MdChurch,
+    new OrbitControlTarget(new THREE.Vector3(5.66, 0.3, -3.58), 5),
+    new THREE.Vector3(5.66, 1.5, -3.7),
+    false,
+    true,
+    'places.cathedral.title',
+    'places.cathedral.subtitle',
+    'places.cathedral.description_html',
+    'https://goo.gl/maps/8NeN5vRJenWTXaUbA',
+    'https://www.catedraldegirona.cat/en/',
+    'https://en.wikipedia.org/wiki/Girona_Cathedral',
+    [
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/1/1c/Girona_Cathedral_2020.jpg/800px-Girona_Cathedral_2020.jpg',
+      'https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/Girona_Cathedral_2020_-_nave.jpg/1280px-Girona_Cathedral_2020_-_nave.jpg',
+    ]
+  ),
+  DEVESA: new Place(
+    'DEVESA',
+    'places.forest.navbar_name',
+    MdForest,
+    new OrbitControlTarget(new THREE.Vector3(-2, 0.3, -3.8), 5),
+    new THREE.Vector3(-2, 0.9, -3.8),
+    false,
+    true,
+    'places.forest.title',
+    'places.forest.subtitle',
+    'places.forest.description_html',
+    'https://goo.gl/maps/TxHvcxuPkRSDxSMi7',
+    'https://patrimoni.gencat.cat/en/collection/la-devesa-park',
+    'https://es.wikipedia.org/wiki/Parque_de_la_Dehesa_de_Gerona',
+    [
+      'https://cdn-patrimoni.atmosfera.net/sites/default/files/styles/slideshows_fitxes/public/parc_devesa.jpg?itok=nG0ytzyL',
+      'https://web.girona.cat/o/si?mh=600&i=/documents/368703/10719717/devesa_3%28mitjana%29.jpg/d7432d69-0042-35e0-be7b-33768c56a6e1?t=1622102055849',
```

Comment:
I think it would be nice to self host the images in the near future, but for now is perfectly okay. It let's us iterate faster and It is also cheaper... lol

Reply:
I was lazy for a first MVP